test/deploy/kubernetes/runtime-pods/components_control_node.yaml

```diff
@@ -102,14 +102,14 @@ spec:
           mountPath: /share
         - name: local-node1
           mountPath: /etc/cortx
-      - name: cortx-bg-producer
+      - name: cortx-bg-scheduler
```

Comment:
The format for container names should be: `<component>-<service>`
For example: `s3-bgscheduler`

Reply:
in Chirag's Pr we tried to follow services framework format , and they have cortx-<component>-<services> 
should we revert back to <component>-<service> then?

Reply:
Lets take it as part of redefined pods infra...